lang/tr.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} geçersiz.",
 	"components.form-element.defaultFieldLabel": "Alan",
 	"components.form-element.input.email.typeMismatch": "E-posta geçerli değil",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Sayı daha küçük {max} olmalıdır.} other {Sayı daha küçük veya eşit {max} olmalıdır.}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Sayı daha büyük {min} olmalıdır.} other {Sayı daha büyük veya eşit {min} olmalıdır.}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Sayı {min} değerinden büyük ve {max} değerinden küçük olmalıdır.} other {Sayı {min} değerinden büyük ve {max} değerinden küçük veya bu değere eşit olmalıdır.}}} other {{maxExclusive, select, true {Sayı {min} değerinden büyük veya bu değere eşit olmalıdır ve {max} değerinden küçük olmalıdır.} other {Sayı {min} değerinden büyük veya bu değere eşit olmalıdır ve {max} değerinden küçük veya bu değere eşit olmalıdır.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Sayı {max} değerinden küçük olmalıdır.} other {Sayı {max} değerinden küçük veya bu değere eşit olmalıdır.}}",
```

Comment:
Before `The number must be smaller {max}.` After `The number must be less than {max}.` After seems good.

Before `The number must be less than or equal to {max}.` After is same.